### ISSUE TITLE
Add pages to list

### ISSUE THREAD

**Issue by stefnotch** [tuwien2020/tgi-pages#10]
- [x] https://simulator.io/board
- [x] https://kmio.de/logikrechner.html
- [x] https://logigator.com/
- [x] https://github.com/rwutscher-tgm/Micro16_expanded_compiler
- [ ] ~#52~ (not until it is done and tested)

**Comment by stefnotch** [comment]
Here is another interesting page
https://deepdaydreams.github.io/algo-theory/index.html

Probably more interesting for Algodat though

**Comment by Fancy11111** [comment]
Very nice page, but you are right, where would we even put it?

**Comment by stefnotch** [comment]
Good enough, let's close this and open a new issue when we find more.
